pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -116,6 +119,30 @@ func (bt *BloomTokenizer) PopulateSeriesWithBloom(seriesWithBloom *SeriesWithBlo
 	} // for each chunk
 }
 
-func (bt *BloomTokenizer) TokenizeLine(line string) []Token {
-	return bt.lineTokenizer.Tokens(line)
+// SearchesForTokenizerAndLine is for taking a given search string (ex: on the read/query path) and returning
+// all the possible tokens, given a tokenizer.
+// This is a multi-dimensional slice where the first slice is the offset into the line, and the
+// second slice is the tokens for that offset.
```

Comment:
This is only true if all of the skip offsets return at least one token. Otherwise, the length of the result will be less than the number of skips+1.

Reply:
yep, I've updated the doc accordingly, good catch